src/view/components/AlphaPickerContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FunctionComponent, useCallback, useEffect, useRef, useState } from 'react';
+import AlphaPicker from '../../components/alphaPicker/alphaPicker';
+import { IQuery } from './type';
+
+type AlphaPickerProps = {
```

Comment:
prefer a interface here

Reply:
also, is more readable using full interface/type names than "I" to represent a interface/type.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/9df95d4fcbea0f59809e56ad4fa4b1c92afe0081

Reply:
Normally i use the full component name plus Props (ex: AlphaPickerContainerProps). This way you have an interface that represents the properties/props of the component in question. If you have the time and you agree with this, you can change. otherwise this can be marked as resolved.